src/models/exclude_filter.rs

```diff
@@ -15,8 +15,8 @@ use getset::Getters;
 use serde_derive::Deserialize;
 
 #[derive(Deserialize, Debug, Clone, Hash, PartialEq, Eq, Getters)]
-pub(crate) struct Constraint {
-  /// Scope in which the constraint query has to be applied
+pub(crate) struct ExcludeFilter {
```

Comment:
Let's rename this struct.
```suggestion
pub(crate) struct Filter {
```

Reply:
Updated accordingly